mapie/classification.py

```diff
@@ -938,8 +902,8 @@ def predict(
                     ).sum(axis=0)
                     prediction_sets = np.stack(
                         [
-                            y_comp > (n + 1 - quantile_)
-                            for quantile_ in self.quantiles_
+                            y_comp > alpha2 * (n - 1)
+                            for alpha2 in alpha_
```

Comment:
Can we use consistent names accross the files ? I think `_alpha` is used everywhere else

Reply:
Correct !